kolibri/plugins/learn/assets/src/views/content-card-grid/index.vue

```diff
@@ -32,8 +42,28 @@
 <script>
 
   import validateLinkObject from 'kolibri.utils.validateLinkObject';
+  import { ContentNodeKinds } from 'kolibri.coreVue.vuex.constants';
+  import some from 'lodash/some';
+  import forEach from 'lodash/forEach';
+  import uiSelect from 'keen-ui/src/UiSelect';
   import contentCard from '../content-card';
+
   export default {
+    $trNameSpace: 'contentCardGrid',
+    $trs: {
+      display: 'Display',
+      all: 'All content ({ count, number })',
```

Comment:
Small thing, but could you change this to `{ num, number }` to ensure an exact match with translation strings that have already been translated for 0.5.x? We can change it to `count` in 0.6.x.